applications/nrf5340_audio/src/bluetooth/Kconfig

```diff
@@ -175,7 +175,7 @@ config LOG_BLE_LEVEL
 
 config LOG_AUDIO_SERVICES_LEVEL
 	int "Log level for LE Audio services"
-	default 3
+	default 4
```

Comment:
Do we really want to print all DBG? Shouldn't we instead change those we want to be INF?

Reply:
Let me only update the volume state to INF.